pkg/security/metrics/metrics.go

```diff
@@ -101,6 +101,9 @@ var (
 	// MetricPerfBufferBytesRead is the name of the metric used to count the number of bytes read from a perf buffer
 	// Tags: map
 	MetricPerfBufferBytesRead = newRuntimeMetric(".perf_buffer.bytes.read")
+	// MetricPerfBufferBytesUsed is the name of the metric used to count the percentage of space left in the ring buffer
+	// Tags: map
+	MetricPerfBufferBytesUsed = newRuntimeMetric(".perf_buffer.bytes.used")
```

Comment:
`Used` feels like if they can't be reused ... what about `in_use` ? Sounds mode like "Datadog" 😅

Reply:
Fixed in latest commit